### DISCUSSION TITLE
Who directed the 1979 sci-fi classic "Alien"?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#595] [Galaxy Trivia] [answered]
**Bonus:** How many direct sequels does it have?

**Comment by Veercodeprog** [comment, marked as answer]
"Alien" is a science fiction horror film released in 1979. The movie was directed by RIDLEY SCOTT and stars Sigourney Weaver, Tom Skerritt, Veronica Cartwright, Harry Dean Stanton, John Hurt, Ian Holm, and Yaphet Kotto. The film follows the crew of the commercial space tug Nostromo who encounter a deadly extraterrestrial creature while returning to Earth.

"Alien" has three direct sequels:

"Aliens" (1986) directed by James Cameron
"Alien 3" (1992) directed by David Fincher
"Alien: Resurrection" (1997) directed by Jean-Pierre Jeunet
There have also been two prequels, "Prometheus" (2012) and "Alien: Covenant" (2017), which are set in the same universe and explore the origins of the Alien creature

**Comment by peterbaccaro** [comment]
The 1979 sci-fi classic "Alien" was directed by **Ridley Scott**.

The 1979 sci-fi classic "Alien" has **three** direct sequels. They are:

"Aliens" (1986), directed by James Cameron.
"Alien³" (1992), directed by David Fincher.
"Alien: Resurrection" (1997), directed by Jean-Pierre Jeunet.

In addition to these direct sequels, the "Alien" franchise has also spawned spin-offs, prequels, and other media adaptations, including "Prometheus" (2012) and "Alien: Covenant" (2017), which are prequels to the original "Alien" film, as well as various comics, novels, and video games set in the "Alien" universe.
